config/TestCase222.xml

```diff
@@ -0,0 +1,63 @@
+<?xml version="1.0"?>
+
```

Comment:
review how you are writing the test case file. looks like there are newline characters saved in which are unnecessary